src/universalSortSecondary.java

```diff
@@ -1,90 +1,72 @@
 package components.universalSort;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Objects;
+
 /*
  * Layered implementations of secondary methods for {@code universalSort}
  */
-public abstract class universalSortSecondary implements universalSort {
-    /*
-     * Public members ---------------------------------------------------------
-     */
+public abstract class UniversalSortSecondary implements UniversalSort {
+
+    public ArrayList<String> list;
+
+    // Standard Methods
 
-    /*
-     * Tracks if this is sorted alphabetically
-     *
-     * @updates this
-     *
-     * @ensures {@code this} = true or {@code this} = false
-     */
     @Override
-    public boolean isSortedAlpha = false;
+    public void clear() {
+        this.clear();
+    }
 
-    /*
-     * Tracks if this is sorted numerically
-     *
-     * @updates this
-     *
-     * @ensures {@code this} = true or {@code this} = false
-     */
     @Override
-    public boolean isSortedNum = false;
+    public UniversalSortSecondary newInstance() {
+    }
+
+    @Override
+    public void transferFrom(UniversalSort source) {
+        assert source instanceof UniversalSortSecondary : "Violation of: source is UniversalSortArrayList";
+        UniversalSortSecondary localSource = (UniversalSortSecondary) source;
+        this.list = localSource.list;
```

Comment:
If you need access to this list, it should be accessible or at least mutable through the kernel methods.